openmdao/error_checking/tests/test_check_config.py

```diff
@@ -81,13 +77,10 @@ def test_dataflow_1_level(self):
         # Conclude setup but don't run model.
         p.final_setup()
 
-        warnings = testlogger.get('warning')
-        info = testlogger.get('info')
-        self.assertEqual(len(warnings), 1)
-        self.assertEqual(len(info), 1)
-
-        self.assertEqual(info[0], "The following groups contain cycles:\n   Group '' has the following cycles: [['C1', 'C2', 'C4']]\n")
-        self.assertEqual(warnings[0], "The following systems are executed out-of-order:\n   System 'C3' executes out-of-order with respect to its source systems ['C4']\n")
+        self.assertTrue(testlogger.contains_regex('info',"The following groups contain cycles.*"))
+        self.assertTrue(
+            testlogger.contains_regex('warning',
+                                      "The following systems are executed out-of-order.*"))
```

Comment:
seems like these were better tests when the names were checked as well... is there a reason these were dropped from the tests?

Reply:
I didn't realize that they needed to be specific. Will put that back in. Thanks